### DISCUSSION TITLE
GitHub Page not Playing Audio!!

### DISCUSSION THREAD

**Discussion by Dev-Jago** [community/community#88495] [Other Feature Feedback, Questions, & Ideas] [answered]
### GitHub Pages not Playing Audio

Hello!

I am new to programing and just recently started uploading my first repositories to GitHub. Unfortunately today, I encountered a problem when I created a GitHub page for a virtual piano I coded in html and javascript. The problem is the same aplication runs perfectly on VS Code's live preview, so I have no Idea what to do to debug this.
Anyway, if anyone can help me figure out what is going on or look into my code I would be grateful.
Here is a link to my repository -> https://github.com/Dev-Jago/Custom_Virtual_Keyboard.git

**Comment by Dev-Jago** [comment]
I just swapped the whole script file with someone else's who did the same course but it did not make a difference. Their web page was playing properly tho, so I think it must be something wrong with the HTMl

**Comment by seyLu** [comment, marked as answer]
Hi @Dev-Jago,

[URLs are generally case-sensitive](https://www.w3.org/TR/WD-html40-970708/htmlweb.html#h-4.1). 
![image](https://github.com/community/community/assets/98249191/6557502c-d3b5-4a21-8412-c5d372532dd3)

So to access the 'Audio' directory:

```diff
const playTune = (key) => {
-    const audioSrc = `src/audio/${key}.wav`;
+    const audioSrc = `src/Audio/${key}.wav`;
```

Proof:

https://github.com/community/community/assets/98249191/6b36f6c4-5df8-4a78-ae40-b15c2a9fa90e

**Comment by Dev-Jago** [comment]
Oh mano, you are my angel!! I cannot describe how thankfull I am! As a beginner theese small projects have a great meaning to me and I so happy you went out your way to help me out! My code works now and I am really proud of it, thanks to you! 

Cheers from Brazil seylu!!!
